src/app/pages/reports/sales/TicketReport/TicketReport.tsx

```diff
@@ -0,0 +1,123 @@
+import {CurrencyCode, CurrencyId, currencies} from '../../../../../types/Currency.types'
+import {useScrutinyDetail} from '../../../../components/Cards/ScrutinyDetail/components/ScrutinyDetailTable.hook'
+import TicketDetail from '../../../../components/Modals/TicketDetail/TicketDetail'
+import TicketReportTable from '../../../../components/Tables/Reports/Ticket/TicketReportTable'
+import {useTicketReport} from './TicketReport.hook'
+
+const TicketReport = () => {
+  const {
+    ticketReportUsdState,
+    resetUsdFilters,
+    setTempFiltersUsd,
+    isLoading,
+    setTicketReportParams,
+    handleUsdFilterChange,
+    tempFiltersUsd,
+    setUsdTicketId,
+    selectedTab,
+    setSelectedTab,
+    handleVesFilterChange,
+    resetVesFilters,
+    setTempFiltersVes,
+    setVesTicketId,
+    tempFiltersVes,
+    ticketReportVesState,
+    isLoadingVes,
+  } = useTicketReport()
+  const {
+    ticketDetailState,
+    isTicketDetailLoading,
+    handleCloseTicketModal,
+    refreshCount,
+    setTicketModalShow,
+  } = useScrutinyDetail(ticketReportUsdState.ticketId ?? '')
+  const ticketDetailId = ticketReportUsdState.ticketId || ticketReportVesState.ticketId
+
+  return (
+    <>
+      <ul className='nav nav-tabs' role='tablist'>
+        {currencies.map((currency) => (
+          <li className='nav-item' role='presentation' key={`currency-tab-${currency.currencyId}`}>
+            <button
+              className={`nav-link ${selectedTab === currency.currencyId ? 'active' : ''}`}
+              id={`tab-${currency.currencyCode.toLowerCase()}-tab`}
+              data-bs-toggle='tab'
+              data-bs-target={`#tab-${currency.currencyCode.toLowerCase()}`}
+              type='button'
+              onClick={() => setSelectedTab(currency.currencyId)}
+              role='tab'
+              aria-controls={`tab-${currency.currencyName.toLowerCase()}`}
+              aria-selected={selectedTab === currency.currencyId ? 'true' : 'false'}
+            >
+              Reporte tiquetes en {currency.currencyName}
+            </button>
+          </li>
+        ))}
+      </ul>
+      <div className='tab-content'>
+        {currencies.map((currency) => (
```

Comment:
Create a variable with useMemo, to avoid render this multiples times

const currenciesMemo = useMemo(() => {}, [currencies])